clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/client/OpenstackClientProvider.groovy

```diff
@@ -52,6 +59,118 @@ abstract class OpenstackClientProvider {
   }
 
   /**
+   * Associate an instance with a load balancer.
+   * @param lbPoolId
+   * @param instanceId
+   */
+  void registerInstancesWithLoadBalancer(String lbPoolId, String instanceId) {
+
+    //weight of member
+    int weight = 1
+
+    //get pool
```

Comment:
Global comment ... This type of code commenting is a code smell to me ... I think we should only comment things that need commenting ... These all seem readable and don't require in-line comments.  This was also the feeling from netflix and was reflected on a code comment I received.